utils/yearContext.js

```
@@ -0,0 +1,32 @@
+        return this.yearsIndex.years[this.index]
+    }
+    get previous() {
+        if (this.yearsIndex.years.length > 1) {
+            return new YearContext(this.yearsIndex, this.index - 1)
+        }
+    }
+    get next() {
+        if (this.yearsIndex + 1 < this.yearsIndex.years.length) {
+            return new YearContext(this.yearsIndex, this.index + 1)
+        }
+    }
```

Biome (1.9.4)

[error] This getter should return a value.



(lint/suspicious/useGetterReturn)